CONTRIBUTING.md

```diff
@@ -42,18 +42,18 @@ Before committing, it's a good idea to run `goimports -w .`.
 
 ## Building and Testing
 
-The [Makefile](./Makefile) contains the targets to build, lint, vet and test:
+The [Makefile](./Makefile) contains the targets to build, lint, and test:
```

Comment:
Changes in this file are cleanup unrelated to the MetalLB issue; I can break this file out into a separate PR if needed.